src/libraries/System.Runtime.Numerics/src/System/Numerics/BigIntegerCalculator.PowMod.cs

```diff
@@ -531,7 +541,7 @@ private const
                 power >>= 1;
             }
 
-            return result;
+            return result.Slice(0, resultLength);
```

Comment:
Same here, this was not changed by https://github.com/dotnet/runtime/pull/74112.